platforms/substrate/charts/README.md

```diff
@@ -64,8 +64,8 @@ node:
   ...
```

Comment:
Instructions to update a value file here should not be given, the value files should already have the right values